src/components/notification/NotificationComponent.js

```diff
@@ -0,0 +1,32 @@
+import "./notification.css";
+import { useState, useEffect } from "react";
+import { ImCross } from "react-icons/im";
+import moment from "moment";
+import { updateViewNotification } from "../../module/actions/NotificationAction";
```

Comment:
this should be `import { updateViewNotification } from "../../module/actions/notificationAction";`